spell.ignore

```diff
@@ -453,3 +453,99 @@ dm
 boolean
 inspektor
 subtests
+lucas
+meneghel
+rodrigues
+lukas
+doktor
+ruda
+moura
+amador
+pahim
+rosa
```

Comment:
nice section :-) (did you consider adding `# authors` section perhaps on top of this file?)

Reply:
(I mean that is not a request, it's just a finishing touch which is not really needed)

Reply:
No, I didn't consider, but now that you mentioned, sounds like a good touch. I will send a V2 including that. Thanks.

Reply:
As we discussed let's keep it this way